### PR TITLE
refact(script): refactored the kubera deploy script

### PR DESCRIPTION
**Pull request by w3aman**
Signed-off-by: Aman Gupta <aman.gupta@mayadata.io>

This PR intends to refactor TCID-DIR-INSTALL-ON-LOCAL-HP testcase installation script to include Kubera installation via both official Kubera charts and kubera-charts repo.

### CODE REVIEW

File: stages/infra-setup/TCID-DIR-INSTALL-ON-LOCAL-HP [hunk 1]
```diff
@@ -41,22 +43,51 @@ node() {
...
+  sed 's|release: .*|release: '$RELEASE'|' -i ./values.yaml
+  # Update URL for accessing Kubera
+  sed 's|url: .*|url: '$DOP_URL'|' -i ./values.yaml
+  # Update max member count in a project to 100
+  sed 's|maxMemberCountInOneProject: .*|maxMemberCountInOneProject: 100|' -i ./values.yaml
+  # Enable openebs RC builds deployment
+  sed 's|openebsRCEnable: .*|openebsRCEnable: true|' -i ./values.yaml
+  # Disable nginx-ingress deployment as it is predeployed in Rancher
+  ## Sed command to replace the third line next to "nginx-ingress:"
+  sed '/nginx-ingress:/!b;n;n;n;c\  enabled: false' -i ./values.yaml
```
Comment (harshshekhar15) [inline stages/infra-setup/TCID-DIR-INSTALL-ON-LOCAL-HP:69]: Remove these lines as nginx is not pre-deployed in Konvoy.

Comment (harshshekhar15) [reply stages/infra-setup/TCID-DIR-INSTALL-ON-LOCAL-HP:69]: Set `type.installOpenebs=false` in the values.yaml here as openebs is predeployed in cluster1 in Konvoy.

File: stages/infra-setup/TCID-DIR-INSTALL-ON-LOCAL-HP [hunk 2]
```diff
@@ -41,22 +43,51 @@ node() {
...
-  
   # Add kubera in local repository
   helm repo add kubera https://charts.mayadata.io/
-
   # Apply helm chart
-  helm install --name kubera kubera/kubera-charts --set type.installOpenebs=false --set server.url=$DOP_URL --set server.release=$RELEASE --set server.maxMemberCountInOneProject=100
-
-  # Pass these values as well if need to run ingress as deployment:  --set nginx-ingress.controller.kind=Deployment --set nginx-ingress.controller.service.enabled=true
-
+  helm install --name kubera kubera/kubera-charts --set server.url=$DOP_URL --set server.release=$RELEASE --set server.maxMemberCountInOneProject=100 --set nginx-ingress.enabled=false --set server.openebsRCEnable=true
```
Comment (harshshekhar15) [inline stages/infra-setup/TCID-DIR-INSTALL-ON-LOCAL-HP:83]: Remove `--set nginx-ingress.enabled=false` and add `--set type.installOpenebs=false`

File: stages/infra-setup/TCID-DIR-INSTALL-ON-LOCAL-HP [hunk 3]
```diff
@@ -91,7 +122,7 @@ node() {
 }
 
 if [ "$1" == "node" ];then
-  node $2 $3 $4 $5 $6 $7 $8 $9
+  node $2 $3 $4 $5 $6 $7 $8 $9 ${10}
```
Comment (harshshekhar15) [inline stages/infra-setup/TCID-DIR-INSTALL-ON-LOCAL-HP:125]: Add ${11} as we are passing two new variables.

**Review by gprasath — APPROVED** [review]

**Pull request merged**